diff --git a/tests/test_utility.py b/tests/test_utility.py
index 844a6ec..f769382 100644
--- a/tests/test_utility.py
+++ b/tests/test_utility.py
@@ -37,7 +37,7 @@ class TestUnidecodeUtility(unittest.TestCase):
 
     def test_encoding_error(self):
         f = temp(self.TEST_UNICODE.encode('sjis'))
-        out, err, rc = run(['-e', 'utf8', f.name])
+        out, err, rc = run(['-e', 'utf-8', f.name])
 
         # Text after : ... can differ between Python versions
         self.assertRegex(err, '^Unable to decode input line ')
